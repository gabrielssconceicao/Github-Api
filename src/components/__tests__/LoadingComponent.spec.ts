import { describe, expect, it } from 'vitest';
import { mount } from '@vue/test-utils';
import Loading from '../LoadingComponent.vue';

describe('<Loading />', () => {
  it('should render', () => {
    const wrapper = mount(Loading);
    expect(wrapper.exists()).toBe(true);
  });

  it('should match snapshot', () => {
    const wrapper = mount(Loading);
    expect(wrapper.element).toMatchSnapshot();
  });
});
