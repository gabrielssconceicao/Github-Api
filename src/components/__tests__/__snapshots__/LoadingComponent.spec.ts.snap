// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Loading /> > should match snapshot 1`] = `
<div
  class="w-[50vw] h-[50vw] border-2 border-t-transparent border-white rounded-full animate-spin"
/>
`;
